ftpretty.py

```diff
@@ -257,6 +296,16 @@ def mkdir(self, new_dir):
         """ Create directory on the server """
         return self.conn.mkd(new_dir)
 
+    def touch(self, name):
```

Comment:
Could you accomplish the same thing by putting a file with no contents, without needing to create a file locally.

ie.
f.put(None,  'someremote/file/greeting.txt', contents='')

Reply:
It appears that I have completely overlooked that. I've made the changes accordingly.